src/components/Station.vue

```diff
@@ -12,11 +12,25 @@
 
   <div class="input-group mb-3">
     <span class="input-group-text">Parameter</span>
-    <select multiple class="form-control" v-model="parameters">
-      <option v-for="p in props.parameters" :value="p.name">
-        {{ p.name }} [{{ p.unit }}] ({{ p.long_name }})
-      </option>
-    </select>
+    <div class="form-control no-padding">
+      <select
+        multiple
+        class="form-control no-border-radius"
+        v-model="parameters"
+      >
+        <option v-for="p in props.parameters" :value="p.name">
+          {{ p.name }} [{{ p.unit }}] ({{ p.long_name }})
+        </option>
+      </select>
+      <div>
+        <span
+          v-for="p in props.parameters"
+          v-show="filterSelectedParams(p)"
```

Comment:
```suggestion
          v-show="parameters.includes(p.name)"
```

- Inline this function as it is so short?
- Array.indexOf can be Array.includes.

Reply:
Didn't know that it can be inline. Makes totally sense